backend/TaskTree/Models/Project.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+
+namespace TaskTree.Models
+{
+	[Index(nameof(Id), IsUnique = true)]
```

Comment:
This line is not required, ID generation is handled automatically (see `BaseEntity`), so unique IDs are enforced.